tests/drivers/adc/adc_api/src/test_adc.c

```diff
@@ -209,7 +209,8 @@
 #define ADC_1ST_CHANNEL_ID      0
 #define ADC_1ST_CHANNEL_INPUT   ADC_INPUTCTRL_MUXPOS_SCALEDIOVCC_Val
 
-#elif defined(CONFIG_BOARD_NUCLEO_F091RC) || \
+#elif defined(CONFIG_BOARD_NUCLEO_C031C6) || \
+	defined(CONFIG_BOARD_NUCLEO_F091RC) || \
```

Comment:
Is that required then ?

Reply:
Do you mean that since it is filtered it doesn’t have to compile?

IMO it would be nice if it compiles, but is filtered not to be built during CI.

Reply:
> Do you mean that since it is filtered it doesn’t have to compile?

Yes. To me, this confuses the message and I'm afraid we get some follow up support of users trying to run it (like someone finally deciding to make some clean up in this test, leveraging device tree, for instance)
Besides, since filtered out in CI, we won't be able to detect it doesn't anymore (if a regression on that aspect happens for that reason).